README.md

```diff
@@ -8,27 +8,23 @@
 ```sh
 $ npm install -g standard-changelog
 $ cd my-project
-$ standard-changelog CHANGELOG.md -w
+$ standard-changelog
 ```
 
-This will *not* overwrite any previous changelog. The above generates a changelog based on commits since the last semver tag that match the pattern of a "Feature", "Fix", "Performance Improvement" or "Breaking Changes".
+The above generates a changelog based on commits since the last semver tag that match the pattern of a "Feature", "Fix", "Performance Improvement" or "Breaking Changes".
 
-If you first time use this tool and want to generate all previous changelog, you could do
+If you first time use this tool and want to generate all previous changelog, you can run:
 
 ```sh
-$ standard-changelog -i CHANGELOG.md -w -r 0
+$ standard-changelog -r 0
 ```
 
-This *will* overwrite any previous changelog if exist.
-
 All available command line parameters can be listed using [CLI](#cli) : `standard-changelog --help`.
 
-**Hint:** You can alias your command or add it to your package.json. EG: `"changelog": "standard-changelog -i CHANGELOG.md -w -r 0"`.
-
+**Hint:** You can alias your command or add it to your package.json. EG: `"changelog": "standard-changelog"`.
 
 ## [Recommended workflow](https://github.com/conventional-changelog/conventional-changelog-cli#recommended-workflow)
 
-
```

Comment:
This is my text editor Atom automatically applying formatting to Markdown. Is it worth rolling back? I would need to figure out how to make my Atom less opinionated about Markdown, or it will do this every time I edit the README.md.